domains/pleo2.json

```
@@ -0,0 +1,10 @@
+{
+    "owner": {
+        "username": "pleo2",
+        "email": "leooel23m@gmail.com",
+        "discord": "609837167497314342609837167497314342"
+    },
+    "record": {
+        "CNAME": "https://porfolio-pi-nine.vercel.app/"
```

CNAME can't contain protocol or `/`
```suggestion
        "CNAME": "porfolio-pi-nine.vercel.app"
```